Public/Src/Cache/ContentStore/Interfaces/FileSystem/VfsUtilities.cs

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO;
+using BuildXL.Cache.ContentStore.Hashing;
+using BuildXL.Cache.ContentStore.Interfaces.Sessions;
+using BuildXL.Cache.ContentStore.Interfaces.Utils;
+
+namespace BuildXL.Cache.ContentStore.Interfaces.FileSystem
+{
+    /// <summary>
+    /// Defines utility methods for working with VFS
+    /// </summary>
+    public static class VfsUtilities

```

Comment:
>VfsUtilities [](start = 24, length = 12)

Unit tests?

Reply:
TBD. This is a prototype.

---
In reply to: [298425687](https://github.com/microsoft/BuildXL/pull/519#discussion_r298425687) [](ancestors = 298425687)